src/common/fidgets/FidgetWrapper.tsx

```diff
@@ -73,33 +88,36 @@ export function FidgetWrapper({
       toast.error("Failed to save fidget settings", { duration: 1000 });
     }
     setSaving(false);
+    setEditing(false);
+    setSelectedFidgetID("");
```

Comment:
Call `unselect` here

Reply:
In fact, all of this should be rolled  up into a `onExitEditMode` function that is provided